### PR TITLE
IGVF-1582-update-biosample-pages

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: components/common-data-items.js [hunk 1]
```diff
@@ -254,6 +254,23 @@ export function SampleDataItems({
...
+                key={protocol}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                {protocol}
+              </a>
+            ))}
+          </DataItemList>
+        </>
+      )}
```
Comment (forresttanaka) [inline components/common-data-items.js:273]: If you have more than one, do the URLs end up without separators? Like `https://www.protocols.io/link1https://www.protocols.io/link2https://www.protcols.io/link3`? I feel like this needs SeparatedList.

File: components/common-data-items.js [hunk 2]
```diff
@@ -319,10 +336,10 @@ export function BiosampleDataItems({
           </DataItemValue>
         </>
       )}
-      {classification && (
+      {classifications.length > 0 && (
```
Comment (forresttanaka) [inline components/common-data-items.js:339]: If `classifications` is null, it’ll crash when you do `classifications.length`. You can fix that with `classifications?.length > 0`.

File: components/common-data-items.js [hunk 3]
```diff
@@ -400,8 +427,8 @@ export function BiosampleDataItems({
 BiosampleDataItems.propTypes = {
   // Object derived from the biosample.json schema
   item: PropTypes.object.isRequired,
-  // Classification if this biosample has one
-  classification: PropTypes.string,
+  // Classifications if this biosample has one
```
Comment (forresttanaka) [inline components/common-data-items.js:430]: “has at least one”

**Review by forresttanaka — CHANGES_REQUESTED** [review]
Looks nice. One, but probably two, changes needed.

File: components/common-data-items.js (hunk 2 above)
Comment (khzlin) [reply components/common-data-items.js:339]: even if it's a required property?

File: components/common-data-items.js (hunk 3 above)
Comment (khzlin) [reply components/common-data-items.js:430]: done

File: components/common-data-items.js (hunk 2 above)
Comment (forresttanaka) [reply components/common-data-items.js:339]: Is it required? It should say that in the propTypes, and it shouldn’t have a default value on line 311 above.

Comment (khzlin) [reply components/common-data-items.js:339]: ahh i see it now

File: components/common-data-items.js (hunk 1 above)
Comment (khzlin) [reply components/common-data-items.js:273]: done

File: components/common-data-items.js [hunk 4]
```diff
@@ -257,18 +257,15 @@ export function SampleDataItems({
...
-                rel="noopener noreferrer"
-              >
-                {protocol}
-              </a>
-            ))}
-          </DataItemList>
+          <DataItemValue>
+            <SeparatedList>
+              {item.protocols.map((protocol) => (
+                <a href={protocol} key={protocol}>
```
Comment (forresttanaka) [inline components/common-data-items.js:263]: To be consistent, open the protocol page in a new tab, so you can use the `target` and `rel` attributes you had.

**Review by forresttanaka — CHANGES_REQUESTED** [review]
Just one change.

File: components/common-data-items.js (hunk 4 above)
Comment (khzlin) [reply components/common-data-items.js]: done

**Review by forresttanaka — DISMISSED** [review]

File: pages/in-vitro-systems/[id].js [hunk 5]
```diff
@@ -130,6 +122,22 @@ export default function InVitroSystem({
                     </DataItemValue>
                   </>
                 )}
+                {truthyOrZero(inVitroSystem.growth_medium) && (
```
Comment (forresttanaka) [inline pages/in-vitro-systems/[id].js:125]: `truthyOrZero` is for numeric values, but `growth_medium` is a string. You can simply test for its existence.

**Review by forresttanaka — CHANGES_REQUESTED** [review]
One small change.

File: pages/in-vitro-systems/[id].js (hunk 5 above)
Comment (khzlin) [reply pages/in-vitro-systems/[id].js]: done

**Pull request merged**